src/Murder.Editor/Systems/GenericSelectorSystem.cs

```diff
@@ -414,7 +415,7 @@ private void DrawSelectionTween(RenderContext render)
                     float startAlpha = .9f;
                     Color color = Game.Profile.Theme.Accent * (startAlpha - startAlpha * tween);
 
-                    Vector2 size = _selectionBox + expand * 2;
```

Comment:
can we override the overload operator? I think that's possible with an extension method

Reply:
Not sure, this says no https://stackoverflow.com/questions/172658/operator-overloading-with-c-sharp-extension-methods. But let me know if you find out otherwise